update_sc_name/update_scientific_name.py

```diff
@@ -58,20 +82,28 @@ def main():
                            default="production_processing", required=False)
     argparser.add_argument("--taxonomy", type=int, help="taxonomy id e.g. 9606", required=True)
     argparser.add_argument("--scientific-name", help=f"""new scientific name for taxonomy
-                                                     (put in "" for names having space in between)""", required=True)
+                                                     (put in "" for names having space in between)""", required=False)
     argparser.add_argument("--ref-seq-dir", help="path to reference seq directory", required=True)
+    argparser.add_argument("--ref-seq-dir-name", help="name of the reference directory if it could not be "
+                                                      "derived by normalizing scientific name in table", required=False)
+    argparser.add_argument("--retain-old-dir", action='store_true', default=False,
+                           help="Don't convert old reference dir to a link if its a valid dir for some other taxonomy")
     args = argparser.parse_args()
 
     # update directory name in reference sequence directory
     old_sc_name = get_scientific_name_from_eva(args.taxonomy, args.private_config_xml_file, args.profile)
-    update_path_reference_sequence(args.taxonomy, old_sc_name, args.ref_seq_dir, args.scientific_name)
+    new_sc_name = args.scientific_name or get_scientific_name_from_ensembl(args.taxonomy)
+    logger.warning(f"old scientific name: {old_sc_name}, new scientific name: {new_sc_name}")
```

Comment:
This should be an `info` rather than a `warning`

Reply:
updated